#thor:
# Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 125

# Publish all sensor data -----------------------------------
  imu_sensor_controller:
    type: imu_sensor_controller/ImuSensorController
    publish_rate: 125

  force_torque_sensor_controller:
    type: force_torque_sensor_controller/ForceTorqueSensorController
    publish_rate: 125
